invoices/templates/snippets/last_week_report.html

```diff
@@ -0,0 +1,37 @@
+<h2>Changes since last week</h2>
```

Comment:
These seem to be almost exact copies of per-invoice report pages. If that is the case, please split common part to a separate file and include it.